src/lib/Libwin/passwd.c

```diff
@@ -2566,14 +2569,15 @@ unmap_unc_path(char *path)
 static char *
 getAssignedHomeDirectory(char *user)
 {
+	static	char	*id = "getAssignedHomeDirectory";
```

Comment:
Same as above, don't use id (and for the rest of the functions)

Reply:
Hi Bhroam, sorry for the trouble! Looks like I made a mistake; I combined some of my changes with an older version of this file (probably from my outdated master branch). I'll revert the id changes.